src/index.js

```diff
@@ -43,6 +44,9 @@ export default function algoliaAlexaAdapter (opts) {
   skill.handler = function(event, context) {
     const alexa = Alexa.handler(event, context);
     alexa.appId = alexaAppId;
+    if (languageStrings) {
+      alexa.resources = languageStrings;
```

Comment:
@Haroenv I don't think so--we want the value that's been set exactly as it is on line 18.